pkg/plugins/manager/testdata/cdn/plugin/plugin.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
NIt: We could probably get away with re-using one of the existing plugin.json files here, no? Even if it's not a panel type

Reply:
Unfortunately we need `Info.Version` and the only one that has it is `testdata/valid-v2-signature`, which is a datasource :disappointed: